core/src/main/java/tech/tablesaw/io/ReadOptions.java

```diff
@@ -154,6 +157,10 @@ public boolean ignoreZeroDecimal() {
     return ignoreZeroDecimal;
   }
 
+  public boolean ignoreInvalidRows() {
```

Comment:
Needs a method comment. 

I'm not sure this is a good method name. It seems there are many ways that a row could be invalid, but the behavior is simply to skip (more descriptive than ignore) rows when the row length differs from the expected number of columns (skipRowsWithInvalidColumnCount()?)

The method comment should describe how the 'valid' number of columns is determined. Is it from the header?, What if there is no header (ie, the noHeader() option is used). Add a test that shows that it works as expected when there is no header.